### ISSUE TITLE
server initiated backup in timer_script.sh is broken.

### ISSUE THREAD

**Issue by laapsaap** [grke/burp#297]
When timer_script.sh checks for a backup file in clients root folder, it does this.
```
manual_file="$storage_dir/$client/backup"
```
but it returns /backup/CNAME/CNAME/backup, it should be;

```
manual_file="$storage_dir/backup"
```

**Comment by grke** [comment]
OK, thanks for the report.
I think I have fixed it for the next 2.0.x release now.
You may try it via branch 2.0.21, commit fd79d6f5cfeb333cc274beee7de4dd8872a47163.

**Comment by laapsaap** [comment]
Great thanks, will try 2.0.21!
